code/drasil-code/lib/Language/Drasil/CodeSpec.hs

```diff
@@ -23,7 +23,7 @@ import Utils.Drasil (subsetOf)
 import Control.Lens ((^.))
 import Data.List (intercalate, nub, (\\))
 import qualified Data.Map as Map
-import Data.Maybe (mapMaybe)
+import Data.Maybe (mapMaybe, fromJust, isNothing)
```

Comment:
`fromJust` and `isNothing` are typically "code smells" in Haskell. You should use pattern-matching instead.

Reply:
Removed fromJust and isNothing